utility/SPIDEV/gpio.cpp

```diff
@@ -9,15 +9,20 @@
  *
  *  Patched for filedescriptor catching and error control by L Diaz 2018
  */
-
+#include <linux/gpio.h>
 #include "gpio.h"
 #include <stdlib.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/ioctl.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <getopt.h>
```

Comment:
As far as I can tell, we're not using a CLI here. I don't think this include is needed.

Furthermore, I think some other includes here are no longer needed (maybe `stdlib`, `unistd`, and `fcntl`), but I have to look up info on each to find out.

Reply:
resolved this with https://github.com/nRF24/RF24/pull/942/commits/f05236a424247addb210c3125bc7c0996b0cbfe8